drop/modules/mae-pipeline/MAE/Results.R

```diff
@@ -125,6 +134,12 @@ ggplot(melt_dt, aes(variable, value)) + geom_boxplot() +
   labs(y = 'Heterozygous SNVs per patient', x = '') +
     annotation_logticks(sides = "l")
 
+#'
+#' ## Variant Frequency within Cohort Histogram
+ggplot(res,aes(x = cohort_freq)) + geom_histogram( binwidth = 0.02)  +
+  geom_vline(xintercept = maxCohortFreq +.01 ,col = "red") +
+  xlab("Variant frequency in cohort") + ylab("Count")
+
```

Comment:
plot a cohortFreq histogram with a red line for the cutoff value

Reply:
cohort_freq should be unique